workflow/rules/common.smk

```diff
@@ -88,6 +88,14 @@ def compile_output_list(wildcards: snakemake.io.Wildcards):
             for t in get_unit_types(units, sample)
         ]
     )
+    output.append(
+        [
+            "qc/picard_collect_multiple_metrics/%s_%s.%s" % (sample, t, ext)
+            for sample in get_samples(samples)
+            for t in get_unit_types(units, sample)
+            for ext in config.get("picard_collect_multiple_metrics", []).get("output_ext", [])
```

Comment:
should we maybe set a default value here? config schema doesn't say that output_ext is required.

Reply:
see comment above.